kube/services/jobs/indexd-single-table-migration-job.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: indexd-single-table-migration
+spec:
+  template:
+    metadata:
+      labels:
+        app: gen3job
+    spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            preference:
+              matchExpressions:
+              - key: karpenter.sh/capacity-type
+                operator: In
+                values:
+                - on-demand
+          - weight: 99
+            preference:
+              matchExpressions:
+              - key: eks.amazonaws.com/capacityType
+                operator: In
+                values:
+                - ONDEMAND
+      automountServiceAccountToken: false
+      volumes:
+        - name: config-volume
+          secret:
+            secretName: "indexd-secret" # pragma: allowlist secret
+        - name: "indexd-creds-volume"
+          secret:
+            secretName: "indexd-creds" # pragma: allowlist secret
+        - name: config-helper
+          configMap:
+            name: config-helper
+        - name: creds-volume-bk
+          secret:
+            secretName: "indexd-bk-creds" # pragma: allowlist secret
```

Comment:
i think it's `indexd-creds` without "bk"

Reply:
oh ok, that makes sense 👍 could you also add a link to the doc at the top of this file?